Exercise_C/index.css

```diff
@@ -2,3 +2,28 @@
 This css sheet is linked to index.html
 You can style tags, classes (.) or ids (#) from this file.
 */
+.sec{
+    height: 300px;
+    width: 500px;
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+}
+.blue{
+    background-color: blue;
```

Comment:
Hey Arran, this code works well, but conventionally it could be better:

height/ width properties are repeated throughout classes